app/Repositories/Group/GroupRepository.php

```diff
@@ -0,0 +1,507 @@
+
+                $invited_users[] = [
+                    'invited_email' => $user['email'],
+                    'group_id' => $group->id,
+                    'token' => $token
+                ];
+            }
+        }
+
+        foreach ($invited_users as $invited_user) {
+            $this->invitedGroupUserRepository->create($invited_user);
+        }
+
+        $group->projects()->sync($data['projects']);
+
+        event(new GroupCreatedEvent($group, $data['projects'], $assigned_users));
+
+        $this->updateGroupProjectUser($group, $data['projects'], $valid_users);
+    }
+
+    /**
+     * Invite user to the group
+     *
+     * @param $group
+     * @param $user
+     */
+    public function inviteToGroup($group, $user)
+    {
+        $auth_user = auth()->user();
+        $token = Hash::make((string)Str::uuid() . date('D M d, Y G:i'));
+        $group->users()->attach($user, ['role' => 'collaborator', 'token' => $token]);
+        $user->notify(new InviteToGroupNotification($auth_user, $group, $token));
+    }
+
+    /**
+     * Invite members to the group
+     *
+     * @param $group
+     * @param $data
+     * @return bool
+     */
+    public function inviteMembers($suborganization, $group, $data)
+    {
+        $auth_user = auth()->user();
+
+        $invited = true;
+        foreach ($data['users'] as $user) {
+            $con_user = $this->userRepository->findByField('email', $user['email']);
+            $note = array_key_exists('note', $data) ? $data['note'] : '';
+
+            if ($con_user) {
+                $token = Hash::make((string)Str::uuid() . date('D M d, Y G:i'));
+                $group->users()->attach($con_user, ['role' => 'collaborator', 'token' => $token]);
+                $con_user->notify(new InviteToGroupNotification($auth_user, $group, $token, $note));
+            } elseif ($user['email']) {
+                $token = Hash::make((string)Str::uuid() . date('D M d, Y G:i'));
+                $temp_user = new User(['email' => $user['email']]);
+
+                // added org invitation for outside users
+                $inviteData['role_id'] = 3;
+                $inviteData['email'] = $user['email'];
+                $inviteData['note'] = $note;
+                $invited = $this->organizationRepository->inviteMember($auth_user, $suborganization, $inviteData);
+                // ended org invitation for outside users
+
+                // $temp_user->notify(new InviteToGroupNotification($auth_user, $group, $token, $note));
+
+                $invited_user = array(
+                    'invited_email' => $user['email'],
+                    'group_id' => $group->id,
+                    'token' => $token,
+                );
+                $result  = $this->invitedGroupUserRepository->create($invited_user);
+                if(!$result)
+                    $invited = false;
+            } else {
+                $invited = false;
+            }
```

Comment:
Need a space after 'if' keyword.

Reply:
Added a space after 'if' keyword and moved condition into block in GroupRepository@inviteMembers().